tools/project_manager/external_catalog_search.php

```diff
@@ -108,11 +108,14 @@ function do_search_and_show_hits()
     $id = yaz_connect($external_catalog_locator, ["charset" => "UTF-8"]);
     yaz_syntax($id, "usmarc");
     yaz_element($id, "F");
-    yaz_search($id, "rpn", trim($fullquery));
+    yaz_search($id, "rpn", trim(str_replace("\n", " ", $fullquery)));
     $extra_options = ["timeout" => 60];
     yaz_wait($extra_options);
     $errorMsg = yaz_error($id);
 
+    echo "<b>Raw query:</b>";
```

Comment:
Leftover debugging?

Reply:
Feature request from Sharon in task 1900.

Reply:
Well, more or less, anyway. 😁 

If a PM creating a project has to wade 10 pages of results to find the record they want, it's a useful reminder. Most PMs are probably not going to know what `@attr 1=1003` means, but it's pretty easy to figure out.

I also really like seeing the media type the record is for. One of my tests, the first three or four hits were all either sound recordings or video of some kind.

Reply:
I defer to Sharon as someone who actually uses this. To me this feels like it will generate more confusion than value for the majority of our PMs.

Reply:
If you want to hold off merging for a bit, I'll ask Rick what he thinks. If he doesn't like the idea, we can ditch it. I don't want to make things harder for PMs.

Reply:
No rush. Let's figure out what we're all happy with.

One possible alternative would be to hide the raw query in a `<details>` element that's closed by default.